apps/frontend/src/libs/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,32 @@
+import { SIDEBAR_ITEMS } from "~/libs/constants/constants.js";
+import { useLocation } from "~/libs/hooks/hooks.js";
+
+import { SidebarLink } from "./libs/components/components.js";
+import styles from "./styles.module.css";
+
+const Sidebar: React.FC = () => {
+	const { pathname } = useLocation();
+
+	return (
+		<div className={styles["sidebarContainer"]}>
```

Comment:
I think we should use class name like sidebar-container. Because this naming style is already used in other files